lib/text/ttml_text_parser.js

```diff
@@ -550,6 +591,17 @@ shaka.text.TtmlTextParser.addStyle_ = function(
     cue.fontStyle = Cue.fontStyle[fontStyle.toUpperCase()];
   }
 
+  if (imageElement) {
+    let backgroundImageType = imageElement.getAttribute('imagetype');
+    let backgroundImageEncoding = imageElement.getAttribute('encoding');
+    let backgroundImageData = imageElement.textContent;
+    if (backgroundImageType === 'PNG' &&
```

Comment:
Our style guide does not allow the use of `===` unless absolutely required (e.g. telling `null` and `undefined` apart). Please change this to `==`.